### ISSUE TITLE
demo is only for short answers and not long answers?

### ISSUE THREAD

**Issue by search4mahesh** [see--/natural-question-answering#1]
Hi,

demo is only for short answers and not long answers?

Thanks
Mahesh

**Comment by see--** [comment]
Yes, the long answer logits are in `outputs[2]`. You can use the same `tf.argmax` approach to print them in the demo. For the full inference, you can use this [Kaggle Notebook](https://www.kaggle.com/seesee/submit-full).

**Comment by search4mahesh** [comment]
Great repo. 
from (c, s, e, t):
what about answer type (t) like 'short', 'Long', 'Yes', "No", and "no-answer."

I think outputs[2] is for answer type. Please correct if I am mistaken.

Thanks
Mahesh

**Comment by see--** [comment]
* `outputs[:2]` is for short answers
* `outputs[2]` for long answers (not answer type)

You can find similar questions in my post: https://www.kaggle.com/c/tensorflow2-question-answering/discussion/127333
